### DISCUSSION TITLE
What is the recommended way to switch HTTPS certificate verification?

### DISCUSSION THREAD

**Discussion by chubin** [earthboundkid/requests#43] [Support] [answered]
What is the recommended way to switch HTTPS certificate verification?
Should I just use my own transport or is there a standard option for that?

**Comment by earthboundkid** [comment, marked as answer]
You should write your own transport. It seems like users of this library are interested in certificate verification, but it's not something I need personally, so there's nothing built in at the moment.

**Reply by earthboundkid** [reply]
This is in the Wiki now: https://github.com/carlmjohnson/requests/wiki#validate-certificates
